src/styles/movieDetails.module.css

```diff
@@ -0,0 +1,145 @@
+.container {
+  display: flex;
```

Comment:
Height and margin properties are duplicated here. Same applies for other classes here in this file and other css files in the project.